test/.env

```diff
@@ -2,7 +2,7 @@
 # shellcheck disable=SC2034,SC2154
 
 # Current container image
-INFIX_TEST=ghcr.io/kernelkit/infix-test:1.9
+INFIX_TEST=ghcr.io/kernelkit/infix-test:2.0
```

Comment:
This was a good idea!  If it had been me, I'd probably have bumped it to 1.10, but since we're upgrading a major component in the image, we should probably also bump the major number of the image.

Unless @mattiaswal and @wkz disagree, I think we should update the test/docker/README.md with this recommendation for the future.

Reply:
I already pushed 2.0, i think we should change this for the future.